### PR TITLE
337 various fixes

### PR DESCRIPTION
**Pull request by enriquesouza**
A few more small fixes from the issue 337

### CODE REVIEW

File: voltixApp/VoltixApp/Views/Vault/VaultPairDetailView.swift [hunk 1]
```diff
@@ -55,7 +55,11 @@ struct VaultPairDetailView: View {
                     .frame(maxWidth: .infinity, alignment: .center)
                 
                 ForEach(devicesInfo, id: \.Index) { device in
-                    VaultPairDetailCell(title: device.Signer, description: .empty)
+                    if device.Signer == Utils.getLocalDeviceIdentity() {
```
Comment (johnnyluo) [inline voltixApp/VoltixApp/Views/Vault/VaultPairDetailView.swift:58]: This is not correct, instead you should compare `device.Signer` , `vault.localPartyID`

File: voltixApp/VoltixApp/Views/Vault/EditVaultView.swift [hunk 2]
```diff
@@ -76,6 +77,10 @@ struct EditVaultView: View {
         }
     }
     
+    var deviceName: some View {
+        Text(Utils.getLocalDeviceIdentity()).padding(.top, 30)
```
Comment (johnnyluo) [inline voltixApp/VoltixApp/Views/Vault/EditVaultView.swift:81]: Here as well , you should not call `Utils.getLocalDeviceIdentity()` after the vault is already created , use `vault.localPartyID` instead

File: voltixApp/VoltixApp/Views/Vault/VaultPairDetailView.swift (hunk 1 above)
Comment (enriquesouza) [reply voltixApp/VoltixApp/Views/Vault/VaultPairDetailView.swift:58]: Changing now

File: voltixApp/VoltixApp/Views/Vault/EditVaultView.swift (hunk 2 above)
Comment (enriquesouza) [reply voltixApp/VoltixApp/Views/Vault/EditVaultView.swift:81]: Changing now

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**